src/client/Client.cc

```diff
@@ -8039,6 +8044,10 @@ bool Client::make_absolute_path_string(Inode *in, std::string& path)
     return false;
   }
 
+  // Make sure this function returns path with single leading '/'
+  if (path.length() && path[0] == '/' && path[1] == '/')
```

Comment:
Why not also do this in a loop just as you did above ?

Reply:
As mentioned above, make_absolute_path_string is a generic function which should return absolute path with single leading '/' as other consumes would expect it. If we add a loop, it would remove all the leading '/' which might not be expected by other consumers.